KakapoExample/KakapoTests/KakapoDBTests.swift

```diff
@@ -361,8 +343,8 @@ class KakapoDBTests: QuickSpec {
                 let users = sut.create(UserFactory.self, number: 100)
                 
                 dispatch_apply(100, dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_BACKGROUND, 0)) { i in
-                    sut.update(users[i])
-                    sut.delete(users[i])
+                    let _ = try? sut.update(users[i])
```

Comment:
Why `let _ =` ? It doesn't return anything, right? Also for test that should never throw I would use `try!` To make sure the test would fail